spec/html_preprocessor_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe PDFKit::HTMLPreprocessor do
```

Comment:
Ah, I hadn't looked too closely. I was relying on GitHub to hide this when it was completed. I'm +1 on this PR at this point then. Feel free to merge @cdwort